src/openms/include/OpenMS/KERNEL/MSChromatogram.h

```diff
@@ -51,30 +51,26 @@ namespace OpenMS
     @brief The representation of a chromatogram.
     @ingroup Kernel
   */
-  template <typename PeakT>
-  class MSChromatogram :
-    private std::vector<PeakT>,
+
+  class OPENMS_DLLAPI MSChromatogram :
```

Comment:
With LTO it's certainly no problem. Without LTO I am pretty sure that it makes (if at all) a neglectable difference. Most time will be most likely be spent on moving the spectrum into the cache.